test/CMakeLists.txt

```diff
@@ -46,3 +46,44 @@ foreach(GTEST_SOURCE_file ${tests})
 
    set_tests_properties(${BINARY_NAME} PROPERTIES TIMEOUT 240)
 endforeach()
+
+find_package(PythonInterp 3 REQUIRED)
+
+if(NOT PYTHONINTERP_FOUND)
+  message(STATUS "No PythonInterp found. Linters will not be executed")
```

Comment:
I think at the `STATUS` level, these messages will not be shown during a `colcon build`, though they would be at a `WARNING` level. I think that is fine, but I wanted to confirm this is the style of verbosity that is preferred

Reply:
Changed https://github.com/ros/console_bridge/pull/73/commits/c8efe0116935a0b6cb9a1f2228e04876c2e3aff5